scripts/ock-cleanup.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+########### generic ###########
+title()
+{
+    case "$1" in
+    "SECTION")
+        shift 1
+        echo "########################################"
+        echo "## $*"
+        echo ""
+        ;;
+    "ENDSECTION")
+        echo ""
+        echo "                                      ##"
+        echo "########################################"
+        echo ""
+        ;;
+    "PARA")
+        shift 1
+        echo ""
+        echo "## $*"
+        ;;
+    "LINE")
+        shift 1
+        echo "$*"
+        ;;
+    *)
+        echo "$*"
+        ;;
+    esac
+}
+
+########### ock-specific ###########
+
+SLOT=${OCK_SLOT:-"3"}
+PIN=${OCK_PIN:-"12345678"}
+
+TMPDIR="tmp-ock"
+unset OPENSSL_CONF
+
+title SECTION "Cleanup (opencryptoki)"
+
+title PARA "Remove directories"
+rm -rf ${TMPDIR}
+
+title PARA "Clean keys"
+
+P11SAK=$(command -v p11sak)
+if [ -z "${P11SAK}" ]; then
+	title LINE "p11sak tool is required"
```

Comment:
You need to say where to find it (not installed on my system and I would have no idea where to pick it)

Reply:
p11sak should be part of the opencryptoki RPM (or will be installed by `make install`, if built from source). I'll change the message accordingly.

Reply:
probably a configure check should require 3.14 min or consider opencryptoki simply not available